plugins/process/src/main/resources/unix-func.sh

```diff
@@ -2,8 +2,8 @@
 
 findpids() {
    PIDS=""
-   for pid in `ls /proc`; do
-      grep /proc/$pid/environ -e "RADARGUN_TAG=$1" > /dev/null 2> /dev/null;
+   for pid in `ps -e -o pid | tail -n +2`; do
+      ps eww $pid | grep "RADARGUN_TAG=$1" > /dev/null 2> /dev/null;
```

Comment:
Eh, I should have looked on the commit message :) On what OS have you met the /proc limitation?